frontend/src/router.js

```diff
@@ -704,35 +704,59 @@ async function requireChecklist(to, from, next) {
 }
 
 export function campFromRoute(route) {
-  return apiStore.get().camps({ id: route.params.campId })
+  if (route.params.campId) {
```

Comment:
here i would prefer a guard clause with early return.
This is whats done here.

if you want to communicate it positively (we need the campId to fetch the camp with the correct id), then
technically, the else can be omitted.
If a function does not return anything, the return value of the function call is undefined.
But you also can communicate it explicitly that this case leads to an undefined.
But you can communicate explicitly, that

Reply:
What do you mean by guard clause?

```cs
export function campFromRoute(route) {
  if (!route.params.campId) {
    return undefined
  }
  return apiStore.get().camps({ id: route.params.campId })
}
```

or

```cs
export function campFromRoute(route) {
  if (route.params.campId) {
    return apiStore.get().camps({ id: route.params.campId })
  }
}
```

Reply:
Variant 1 would be the guard (with early return).

Variant 2 would be just omitting the return undefined implicitly.

Reply:
I can live with both variants - choose one ;)

Reply:
I choose Variant 1

Reply:
done